Sponsoring page: first draft with sponsoring tiers (#364)
OSGeo/grass-website: content/contribute/sponsoring.md

### Why sponsor?

*The GRASS GIS project is a volunteer-based organization*, 
it is an affiliated project and founding member of the
Open Source Geospatial Foundation ([OSGeo](https://osgeo.org/)).

There is no cost to participate in the project itself or to use
the software.

But with our own budget we can produce even better software and
documentation for you. And if you don't have time to contribute,

> maybe a donation makes you feel better when using GRASS GIS in

Reviewer: ```suggestion
maybe a donation will make you feel better about using GRASS GIS in
```

> your company or research. Your financial support will also help
> us set up the needed efforts and infrastructure to ensure
> project and software sustainability in the long run.

Reviewer: ```suggestion
your company or research. Your financial support will also help
us to build up the necessary effort and infrastructure to ensure
the long-term sustainability of the project and the software.
```

<div align="center"><button class="btn btn-primary"><b><a href="https://opencollective.com/grass/contribute" target="_blank">SUPPORT GRASS GIS</a></b></button></div>

### How we use the money

The donations are controlled in full transparency by the GRASS
Project Steering Committee ([PSC](https://trac.osgeo.org/grass/wiki/PSC)).

> Funds offered to GRASS GIS are used to sponsor general GRASS GIS
> efforts, especially:

Reviewer: ```suggestion
Funds offered to GRASS GIS will be used to support general GRASS GIS
efforts, in particular:
```

- the organization of face-to-face coding sessions or [**Community Sprints**](https://grasswiki.osgeo.org/wiki/Category:Code_Sprint), *financial support can be used to make developers' participation possible*
- offering [**Student Grants**](https://grasswiki.osgeo.org/wiki/Student_Grants) to support students while they contribute with actual coding, bug fixing, documentation or creation of educational resources.

Also *bug fixing may be financed* - community polls will identify outstanding problems which cannot be fixed on a voluntary basis;
or specific bug-bounties can be arranged (please [contact us](https://trac.osgeo.org/grass/wiki/PSC#Members) first).

### How to become a sponsor?

We handle all donations and financial support in
[**OpenCollective**](https://opencollective.com/grass/contribute).

> There, you can find different **sponsoring tiers** from which to choose
> and also a **donation tier** that can be customized as you prefer.
> The different sponsoring levels and suggested yearly contribution
> amounts are described below.

Reviewer: ```suggestion
There you will find several **sponsorship tiers** to choose from
and also a **donation tier** which can be customized to suit your needs.
The different sponsorship tiers and suggested annual contributions are
described below.
```

|Tier   	|Amount (USD)  |
|---	    |---	|
|Diamond	|10000  |
|Platinum	|5000  	|
|Gold  	    |2000  	|
|Silver  	|1000  	|
|Bronze  	|500   	|
<br>
<div align="left"><button class="btn btn-primary"><b><a href="https://opencollective.com/grass/contribute" target="_blank">BECOME A SPONSOR</a></b></button></div>

### Sponsorship benefits

> All sponsors are important for the GRASS GIS project, no matter
> what funding level they commit to.
> In recognition of the various levels of sponsor contributions,
> each sponsor receives particular benefits. These are mostly
> limited to logo sizes so far, but as the PSC sees potential,
> we will enlarge the list of benefits according to support level.

Reviewer: ```suggestion
All sponsors are important to the GRASS GIS project, no matter
what level of funding they commit to.
In recognition of the different levels of sponsorship,
each sponsor will receive specific benefits. These are mostly
limited to logo sizes so far, but as the PSC sees potential,
we will extend the list of benefits according to the level of support.
```
